lib/index.js

```diff
@@ -79,16 +80,50 @@ module.exports = {
                             config.validate.params = config.validate.params || {};
                             config.validate.params[validator.parameter.name] = validator.schema;
                             break;
-                        case 'query':
-                            config.validate.query = config.validate.query || {};
-                            config.validate.query[validator.parameter.name] = validator.schema;
-                            break;
                         case 'body':
                         case 'form':
                             config.validate.payload = validator.schema;
                             break;
                     }
                 });
+
+                config.validate.query = function(value, options, callback) {
```

Comment:
Should this be set every time? What if there are no query parameters validators?